src/main/scala/RockPaperScissors.scala

```diff
@@ -0,0 +1,26 @@
+class RockPaperScissors {
+
+  println("Enter choice...")
+  val userInput = scala.io.StdIn.readLine().toLowerCase
+  var random = scala.util.Random
+
+  def computerTurn: String = {
+    random.nextInt(3) match {
+      case 0 => "rock"
+      case 1 => "paper"
+      case 2 => "scissors"
+    }
+  }
+
+  def play = {
```

Comment:
can we rework this please so that it's not this little monster of if statements :)